packages/manager/modules/pci/src/projects/project/ssh-keys/add/index.js

```diff
@@ -0,0 +1,22 @@
+import angular from 'angular';
+
+import '@ovh-ux/manager-core';
+import '@ovh-ux/ng-ovh-api-wrappers'; // should be a peer dependency of ovh-api-services
```

Comment:
Shouldn't be injected to the module dependency list?

Reply:
> // should be a peer dependency of ovh-api-services

So it's ovh-api-services which required it :)